services_app/src/main/java/org/opendatakit/services/MainActivity.java

```diff
@@ -92,6 +92,17 @@ protected void onCreate(Bundle savedInstanceState) {
     }
   }
 
+  @Override
+  public void onBackPressed() {
+    int count = getSupportFragmentManager().getBackStackEntryCount();
+    if (count == 1) {
+      startActivity(new Intent(MainActivity.this,MainActivity.class));
+      startActivity(getIntent());
```

Comment:
Why call startActivity twice in a row? This does not seem like a correct workflow since you are basically changing your lifecycle then immediately changing the lifecycle again.

Reply:
you might have a reason which is why I am asking.

Reply:
I am calling this again as to inflate all the menu items again , if we don't do this then the about menu item will be left invisible